PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRegister.swift

```diff
@@ -0,0 +1,122 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+/// Global singleton responsible to store plugin renderer instances
+@objc public class PrebidMobilePluginRegister: NSObject {
+    @objc public static let shared = PrebidMobilePluginRegister()
+    
+    private let queue = DispatchQueue(label: "PrebidMobilePluginRegisterQueue", attributes: .concurrent)
+    private var plugins = [String: PrebidMobilePluginRenderer]()
+    
+    private let defaultRenderer = PrebidRenderer()
+    
+    private override init() {
+        super.init()
+    }
+    
+    /// Register plugin as renderer
+    @objc public func registerPlugin(_ renderer: PrebidMobilePluginRenderer) {
+        let rendererName = renderer.name
+        
+        queue.async(flags: .barrier) { [weak self] in
+            guard self?.plugins[rendererName] == nil else {
```

Comment:
It's hard to follow the condition of this expression. Can we use `if self?.plugins[rendererName] == nil {` ?